app/src/androidTest/java/com/mapbox/maps/testapp/integration/fragment/MapFragment.kt

```diff
@@ -0,0 +1,64 @@
+package com.mapbox.maps.testapp.integration.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.mapbox.maps.MapView
+import com.mapbox.maps.Style
+
+class MapFragment : Fragment() {
```

Comment:
Can we somehow use same `MapFragment` as in our examples? They look pretty alike and we could make it single source of truth.

Reply:
I actually like the idea of having 2 different implementations. The test app variant calls onDestroy in onDestroyView while this impl uses onDestroy instead.